src/utils.jl

```diff
@@ -54,7 +54,14 @@ function gensubscribername()
     return name
 end
 
-
+DEFAULT_PORT = 6000;
+reset_default_port() = global DEFAULT_PORT = 6000;
```

Comment:
So default port numbering is cool but then its a bit clunky for the user to go to another file which contains a subscriber and get the port of the associated publisher.